web.py

```diff
@@ -213,6 +214,33 @@ def test_ws():
             flash('Event not yet implemented', 'danger')
     return render_template('test_ws.html', recipient=recipient, event=event, events=events)
 
+@roles_accepted(Role.ROLE_ADMIN, Role.ROLE_FINANCE)
+@app.route('/user_kyc', methods=['GET', 'POST'])
+def user_kyc():
+    email = ''
+    if request.method == 'POST':
+        email = request.form['email']
+        user = User.from_email(db.session, email)
+        if user:
+            kycrequest = KycRequest.from_user(db.session, user)
+            if kycrequest.status == kycrequest.STATUS_COMPLETED:
+                token = kycrequest.token
+                pdf = kyc_core.download_pdf_backup(token)
+                if pdf:
+                    response = make_response(pdf)
+                    response.headers.set('Content-Type', 'application/pdf')
+                    return response
+                #else:
```

Comment:
remove dead code

Reply:
apart from the file being downloaded... everything bout it seems to be working.

Reply:
changed some of the host var more specifically the 
```
- b2_account_id 
- b2_application_key
- kyc_bucket
```

but now im getting the following:
```
  File "/home/eoliveros/git/beryllium/venv/lib/python3.9/site-packages/flask/helpers.py", line 191, in make_response
    return current_app.make_response(args)
  File "/home/eoliveros/git/beryllium/venv/lib/python3.9/site-packages/flask/app.py", line 1727, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a BytesIO.
[geventwebsocket.handler INFO] 172.23.96.1 - - [2021-09-30 16:33:41] "POST /user_kyc HTTP/1.1" 500 542 1.840609
```

not sure what this message means.
```